internal/proxy/impl.go

```diff
@@ -2664,8 +2662,7 @@ func (node *Proxy) Search(ctx context.Context, request *milvuspb.SearchRequest)
 		}
 	}()
 
-	log.Debug(
-		rpcReceived(method))
+	log.Debug(rpcReceived(method))
```

Comment:
if we remove travel timestamp, we also remove the functionality of MVCC.
Any functionality relies on this?